src/auto-merge.js

```diff
@@ -19,6 +19,26 @@ module.exports = (robot) => {
     return handler(context)
   })
 
+  safeBind(['status'], context => {
+    const branches = context.payload.branches
+
+    if (branches.length !== 1) {
+      return
+    }
+
+    const [branch] = branches
+    return context.github.pulls.list({...context.repo(), head: `openstax:${branch.name}`}).then(response => {
```

Comment:
Is the org name not available in `context.repo().name`?

Reply:
hmmm... i think `context.repo` would be the base repo in the case of a cross fork PR, so that might not be exactly correct